README.md

```diff
@@ -13,12 +13,24 @@ Create and manage Azure Virtual Machines directly from VS Code.
 
 ### Create New Virtual Machine
 
+- Create an Azure VM with the default image of "Ubuntu 18.04-LTS" and size "Standard D2s V3"
+- Automatically generates and configures an SSH key used to remotely connect to new Azure VM
+
 ![CreateVirtualMachine](resources/CreateVirtualMachine.gif)

```

Comment:
I'm actually not that big a fan of gifs in the readme anymore. In the user study, it seemed like people would ignore gifs (maybe it moves too fast?) and found screenshots more helpful. Curious what @fiveisprime thinks

Also the gifs seem to have a lot of your testing names (naturins3000000, afsdafsadfs, etc.). If we just did screenshots, it would be easier to hide all that junk and display just the stuff that actually matters.

Reply:
Agreed, I'd much rather see some high quality screenshots.

Reply:
Added some rather high quality screenshots.